eng/Packages.Data.props

```diff
@@ -58,9 +58,9 @@
   <ItemGroup Condition="'$(IsClientLibrary)' == 'true'">
 
     <!-- BCL packages -->
-    <PackageReference Update="System.Buffers" Version="4.5.0" />
-    <PackageReference Update="System.Memory" Version="4.5.3" />
-    <PackageReference Update="System.Memory.Data" Version="1.0.1" />
+    <PackageReference Update="System.Buffers" Version="4.5.1" />
+    <PackageReference Update="System.Memory" Version="4.5.4" />
+    <PackageReference Update="System.Memory.Data" Version="1.0.2" />
```

Comment:
These effect the entire repo, was changing these by design?

Reply:
yes, we have to ship with latest core and these things updated. This won't affect master branch nor any other service effectively, we're on the hotfix branch.

Reply:
Oh gotcha, I forgot this is a hotfix branch.